### PR TITLE
COMMERCE-5661 | COMMERCE-5399

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:relvant

**Comment by Commando18** [comment]
ci:test:relevant

File: modules/dxp/apps/commerce/headless/headless-commerce-bom-impl/src/main/java/com/liferay/headless/commerce/bom/internal/dto/v1_0/converter/DiagramDataDTOConverter.java [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+			cpDefinitionDiagramSettings.fetchCPAttachmentFileEntry();
+
+		if (cpAttachmentFileEntry != null) {
+			FileEntry fileEntry = cpAttachmentFileEntry.getFileEntry();
+
+			image = DLUtil.getDownloadURL(
+				fileEntry, fileEntry.getFileVersion(), null, null);
+		}
+
+		String imageURL = image;
```
Comment (dsitu) [inline modules/dxp/apps/commerce/headless/headless-commerce-bom-impl/src/main/java/com/liferay/headless/commerce/bom/internal/dto/v1_0/converter/DiagramDataDTOConverter.java:81]: Just call it imageURL
```suggestion
		String imageURL = StringPool.BLANK;

		CPDefinitionDiagramSettings cpDefinitionDiagramSettings =
			_cpDefinitionDiagramSettingsService.getCPDefinitionDiagramSettings(
				(Long)dtoConverterContext.getId());

		CPAttachmentFileEntry cpAttachmentFileEntry =
			cpDefinitionDiagramSettings.fetchCPAttachmentFileEntry();

		if (cpAttachmentFileEntry != null) {
			FileEntry fileEntry = cpAttachmentFileEntry.getFileEntry();

			imageURL = DLUtil.getDownloadURL(
				fileEntry, fileEntry.getFileVersion(), null, null);
		}
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**